scripts/vcpkg_ports/conduit/CONTROL

```diff
@@ -1,5 +1,5 @@
 Source: conduit
-Version: v0.5.1
+Version: v0.6.0
```

Comment:
I had to upgrade to `conduit@0.6.0` on Windows due to `node::to_string()`
I think this matches our spack version.